source/docs/articles/sites/migrate.md

```diff
@@ -43,98 +36,59 @@ When preparing a site for export, there are a few best practices to follow:
 * Take a look at your codebase and **remove any non-core code from your site** that you aren’t planning on running on Pantheon.
 * If you’ve been using the database for things other than Drupal or WordPress, you should **drop or skip any unnecessary or unrelated database tables** that your site doesn’t need.
 
+
 ### Create Single-file Archives
-If your site can be packaged with a total archived size less than 500MB, then you can import that single file to create your site.
-Drupal archives created with [drush archive dump](http://drushcommands.com/drush-6x/archive/archive-dump) are known to work when imported during site creation.
+Site's that can be packaged with a total archived size less than 500MB are able to use single-file archives during the import process. These archives can be created with [drush](/docs/articles/drupal/prepare-drupal-for-export#create-archive-using-drush) for Drupal sites and  [Plugins](/docs/articles/wordpress/export-an-existing-wordpress-site#export-wordpress-via-plugins) for WordPress.
```

Comment:
@bmackinney I still need to work in what goes into a single-file archive. Do you think that should be here in the main migration doc, or should it be specific to each CMS in their respective export docs?